### ISSUE TITLE
Add functionality to remove songs from Goldify playlist, as well as add songs from top listens table

### ISSUE THREAD

**Issue by MattGrantUMich** [Goldify/Goldify#24]
**Is your feature request related to a problem? Please describe.**
Feature request. This would be one of the last major leaps before we could produce an MVP in my eyes. 

**Describe the solution you'd like**
Have a left-most column on each of the tables for adding and or removing songs from each of the tables.
- NOTE: We would need to signify that you can only REMOVE from the Goldify playlist, or ADD to the Goldify playlist from the Top Listens table
There's probably a really clean way to do this, but an initial mock up should be done and then discussed by the Goldify team.

**Describe alternatives you've considered**
None at the moment, but I may come back and change this.

**Additional context**
Basically, we just need to take the already created utility functions created by Kyle and add it into our Goldify page. This should be relatively easy to do, since a good bulk of the work is already done. After this is complete, though, we should do some real thorough integration testing.
